community/gardening/index.md

```diff
@@ -8,15 +8,29 @@ sidebar:
+
+Elevate your impact with free compute from AWS, SIG projects in Security, Ops, and more, a global-friendly schedule, and updated documentation and training resources.
+
 ## Jump In!
-* Join [Spinnaker Slack](https://join.spinnaker.io) to chat in [event & team channels](slack-map.md). Visit __#gardening-general__, our main event channel to engage.
-* Find out how weeklong Gardening Days 2.0 will work on the [Hackathon logistics](hack-logistics.md) page.
-* The [What to hack](what-to-hack.md) page list all eligible contributions types.
-* Check the [Master schedule](schedule.md) to see our more time-zone inclusive global schedule.
-* The updated [Slack map](slack-map.md) has a shorter, sweeter list of Slack channels to engage in.
-* Set up your [Development environment](dev-environment.md) using this guide.
-* Dive into [Developing plugins](developing-plugins.md) with this training series.
-* Gardening project repositories should live in the [Spinnaker-hackathon Github organization](https://github.com/spinnaker-hackathon).
+- Join [Spinnaker Slack](https://join.spinnaker.io) to chat in [event & team channels](slack-map.md). Have questions? Ask them in [__#gardening-general__](https://spinnakerteam.slack.com/archives/CV4A90DPF), our main event channel.
+- Find out how weeklong Gardening Days 2.0 will work on the [Hackathon logistics](hack-logistics.md) page. Get inspired by these challenges:
+  - **Fix Bugs & Error Messages** _Retrofit 500 Challenge_ - Fix the most commonly occurring 500 errors
+  - **Extend Spinnaker** _Plugin Challenge_ - Work with Armory engineers to customize Spinnaker with a plugin
+  - **Share Content** _Automate Your SDLC Challenge_ - Share your Spinnaker with technical content
+  - **Join a SIG Project** _SIG Challenges_ - Everyone's welcome! Join a SIG team by showing up at a Special Session SIG meeting:
+    - Documentation SIG - **Wednesday July 15, 10:30 AM PDT** - Share docs feedback and develop content for Spinnaker.io
+    - Security SIG - **Thursday July 16, 11 AM PDT** - Discuss and develop threat models for Spinnaker deployment endpoints
+    - Ops SIG - **Thursday July 16, 10 AM PDT** - Build a Spinnaker customization to automate dynamic accounts configuration
```

Comment:
```suggestion
    - Ops SIG - **Thursday July 16, 10 AM PDT**
      Build a Spinnaker customization to automate dynamic accounts configuration.
```

Reply:
Assuming I didn't break the formatting on all these. My suggestion is to separate each main list item from its elaborative sentence using line breaks and indentation instead of space-dash-space. And add a period at the end of those.